rocAL_pybind/rocal_pybind.cpp

```diff
@@ -418,6 +420,11 @@ PYBIND11_MODULE(rocal_pybind, m) {
         .value("FT", ROCAL_FT)
         .value("TF", ROCAL_TF)
         .export_values();
+    py::enum_<RocalOutOfBoundsPolicy>(types_m, "RocalOutOfBoundsPolicy", "Rocal Audio Out Of Bounds Policy")
+            .value("PAD", ROCAL_PAD)
```

Comment:
Check indentation

Reply:
It was messed in the merge, corrected it